nodes/vector/turbulence.py

```diff
@@ -120,27 +120,18 @@ def process(self):
         if verts and verts[0]:
             for vertex in verts[0]:
 
-                # origin
+                # set the offset origin for random seed
                 if rseed == 0:
-                    origin = 0.0,0.0,0.0
-                    origin_x = 0.0
-                    origin_y = 0.0
-                    origin_z = 0.0
+                #we set offset value to 0.0
```

Comment:
a comment directly under an `if` should be indented by 4 spaces.. like the next line